params/chromium-params.md

```diff
@@ -63,3 +64,5 @@ Chromium's implementation assigns the following values:
 [max event-level channel capacity for navigations value]: https://source.chromium.org/chromium/chromium/src/+/main:content/browser/attribution_reporting/attribution_config.h;l=77;drc=f470a177bbf7e636c598fd8c9e9dee4f936e73ad
 [Max event-level channel capacity for event sources]: https://wicg.github.io/attribution-reporting-api/#max-event-level-channel-capacity-per-source
 [max event-level channel capacity for events value]: https://source.chromium.org/chromium/chromium/src/+/main:content/browser/attribution_reporting/attribution_config.h;l=114;drc=22c828d3e18706a79fde266109d0e9290a679c56
+[Max unique attribution scope sets per navigation]: .
```

Comment:
Unique "scope sets," to me means something like
```
Scope set A: {1, 2}
Scope set B: {1, 3}
etc.
```
The above are "unique scope sets" (the uniqueness is asserted by comparing the sets).

I'm confused as to how this applies to the feature. Can someone please help me understand?

Reply:
I missed that this is per navigation -- I think I understand the intention now. This comment can be resolved (I don't have the ability to resolve it).